### DISCUSSION TITLE
not able to create custome logs

### DISCUSSION THREAD

**Discussion by vish1plunker** [markshust/docker-magento#631] [General]
there may be some issue with permission to create logs with set up. not able to figure out how to fix in image. I am trying custom logs and it works fine without docker set up 
any guidance will help

**Comment by rangerz** [comment]
The custom log (`/var/log`) is stored in docker container.

Try this command:
```
bin/cli tail -f ./var/log/custom.log
```

or uncomment this line and restart docker (`bin/restart`)
https://github.com/markshust/docker-magento/blob/4331507f9e252961bc98fc382602b0f8925ab4e4/compose/docker-compose.dev.yml#L16
The log will write into your local folder

**Comment by vish1plunker** [comment]
Perfect ! worked with commenting line in .uml file
